Documentation~/ref_tools.md

```diff
@@ -1,18 +1,24 @@
-
-Some tools are available from the [Tools panel](https://docs.unity3d.com/Manual/UsingCustomEditorTools.html#ToolModesAccessSceneViewPanel) in the Scene view when the selection meets the tool's criterion. For example, if you select a Mesh you created with the [Shape](shape-tool.md) tool, the Edit Shape icon appears in the **Tools** panel, which you can click to activate the Shape tool and edit the bounding box or choose a different shape primitive.
+## Tools
 
 This table lists all the tools available in ProBuilder:
 
-| **Tool:**                       | **Description:**                                             |
-| ------------------------------- | ------------------------------------------------------------ |
-| [Shape](shape-tool.md) tool     | Creates a new Mesh with the Shape component, which defines the Mesh's shape primitive. You define the same bounding box for all shape primitives. <br /><br />**Tip**: To modify the primitive or bounding box after creation, select the Mesh and then click **Edit Shape** on the **Tools** panel in the Scene view. |
-| [Poly Shape](polyshape.md) tool | Creates a new Mesh with the Poly Shape component.<br /><br />**Tip**: To modify the original after creation, select the Mesh and then click  **Edit Poly Shape** on the **Tools** panel in the Scene view. |
-| [Cut](cut-tool.md) tool         | Cuts a sub-face into an existing Mesh face. You determine the shape of the new face by defining points on the Mesh to create the edges of the new face. <br /><br />You can use this tool on any face, regardless of whether you created the Mesh with the [New Shape](shape-tool.md) tool, the [Poly Shape](polyshape.md) tool, or by [probuilderizing](Object_ProBuilderize.md) a Unity Mesh. |
+| **Tool** | **Description** |
+| --- | --- |
+| [New Shape and Edit Shape](shape-tool.md) | Creates a new mesh with the Shape component, which defines the mesh's shape primitive. |
+| [New PolyShape and Edit PolyShape](polyshape.md) | Creates a new mesh with the PolyShape component. You then draw a 2D shape, and extrude it to a 3D shape. |
+| [Cut](cut-tool.md) | Creates a subface in an existing mesh face. You design the shape of the new face by defining points on the mesh, then move the face as you do any other face. <br /><br />You can use this tool on any face, regardless of whether you created the mesh with the [New Shape](shape-tool.md) tool, the [New PolyShape](polyshape.md) tool, or by [probuilderizing](Object_ProBuilderize.md) a standard Unity mesh. |
+
+> **Note:** For documentation on the [Bezier Shape](bezier.md) tool, refer to [Experimental features](experimental.md).
+
+## Active context in the Tools overlay
+
+The ProBuilder tools are available in the **Tools** overlay in the **Scene** view. The tools available depend on the active context: [[link to context]]
```

Comment:
```suggestion
The ProBuilder tools are available in the **Tools** overlay in the Scene view. The tools available depend on the active context: [[link to context]]
```

Reply:
Actually hang on. 
Write "Scene" in bold with a capital S, followed by "view" in lowercase.
That's from the style guide.

Reply:
```suggestion
The ProBuilder tools are available in the **Tools** overlay in the **Scene** view. The tools available depend on the active context: GameObject or ProBuilder.
```

Reply:
Can't link to it - it hasn't been published yet.

Reply:
> Actually hang on.
Write "Scene" in bold with a capital S, followed by "view" in lowercase.
That's from the style guide.

When did we change this?!

Reply:
Please disregard my suggestions for that then.